pkg/driver/publishers/volume.go

```diff
@@ -19,4 +19,8 @@ const (
 	DSDSocketDirectory VolumeType = "DSDSocketDirectory"
 	// DatadogSocketsDirectory is deprecated, use DSDSocketDirectory instead
 	DatadogSocketsDirectory VolumeType = "DatadogSocketsDirectory"
+	// DatadogLibrary mounts a Datadog instrumentation library from an OCI image
+	DatadogLibrary VolumeType = "DatadogLibrary"
+	// DatadogInjectorPreload mounts the ld.so.preload file
+	DatadogInjectorPreload VolumeType = "DatadogInjectorPreload"
```

Comment:
Thinking out loud: Is this something we need in the driver? I kind of like that you have included it so that all volumes can be provided by CSI. But we could technically keep this one as an emptyDir for a single file in the cluster agent?